bitnami/zookeeper/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: zookeeper
-version: 5.19.1
+version: 5.19.2
```

Comment:
Please bump the minor version since you are adding a new parameter for the `probeCommandTimeout`

Reply:
thanks @miguelaeh is now bumped to `5.20.0`